src/grandorgue/OrganDialog.cpp

```diff
@@ -262,6 +262,7 @@ OrganDialog::OrganDialog (GOrgueDocumentBase* doc, wxWindow* parent, GrandOrgueF
 	Load();
 	
 	topSizer->Fit(this);
+	topSizer->SetSizeHints(this);
```

Comment:
After this changes I cann't resize this dialog vertically at all: in opens with the whole wcreen height and resizing works only vertically.

Reply:
The behaviour on windows has not changed
![GrandOrgueWin](https://user-images.githubusercontent.com/11586506/135669214-7561c959-5079-4b53-9d7a-07aeee9c547f.png)

Reply:
I suggest to remove SetSizeHints, but to specify a minimum size 640 in OrganDialog::OrganDialog:

```
(GOrgueDocumentBase* doc, wxWindow* parent, GrandOrgueFile* organfile) :
	wxDialog(parent, wxID_ANY, _("Organ settings"), wxDefaultPosition, wxSize(wxDefaultCoord, 640), wxDEFAULT_DIALOG_STYLE | wxRESIZE_BORDER),
```

Reply:
How strange. It did on my windows computer... No, I don't think we should specify a fixed minimum size that way, it should be dictated by the contents of the sizers. And yes, it was my intention to disallow resizing the dialog too much horizontally, so that one wouldn't run into the issue with your first screenshot above.

Reply:
> No, I don't think we should specify a fixed minimum size that way, it should be dictated by the contents of the sizers.

By what algorithm?

- If all contents must fit the minimal size then scrolling is not necessary
- If allow the resizing behind the ''all fit" size then how to consider the contents?

> And yes, it was my intention to disallow resizing the dialog too much horizontally

horizontally - ok, but I could not reduce the size vertically

> so that one wouldn't run into the issue with your first screenshot above.

This issue doesn't depend on the horizontal size.

Reply:
Scrolling will be necessary if the screen is too small for the content, but if the screen is big enough for the content of the dialog then no scrolling of course is needed and shouldn't be available. And yes, in theory the dialog could be too big horizontally for a certain screen resolution but that's quite unlikely to ever happen.

Well, the original issue is solved, I just found some other side effects that perhaps should be addressed too. I have a new test going on right now.

Reply:
Now it is too big vertically and does not allow resizing. The behaviour without SetSizeHints was better
![OrganSettings69](https://user-images.githubusercontent.com/11586506/135678737-95952bb9-0af3-47ac-a5a4-3a6c000f7071.png)

Reply:
Ok, you might be right about that. Seems like since the behaviour is not consistent on the different platforms we perhaps should be content with just fixing the original issue and leave the rest for now.